src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceControllerTest.kt

```diff
@@ -370,11 +369,10 @@ class LicenceControllerTest {
       put("/licence/id/4/submit")
         .accept(APPLICATION_JSON)
         .contentType(APPLICATION_JSON)
-        .content(mapper.writeValueAsBytes(aSubmitLicenceRequest))
     )
```

Comment:
Just checking I've understood - the submit request knows who it is via the username on the token, so can then lookup the COM who is submitting?  But what happens if this is the first time the COM has interacted with the service and they decide to submit a licence that someone else has created? Will their details be known? Will it look them up and create a COM details row for them?

Reply:
Yep, details get stored on login. If someone is creating a licence on behalf of a COM who has never logged in, the responsible COM is also saved on creation.